README.md

```diff
@@ -39,13 +39,13 @@ _Note: All input types will be checked for existence and checksum matching._
 
 ### Supported Inputs
 
-| File Type | Description |
-| :-------: | ------ |
-| bam | Validate bam/cram/sam using `pysam`. <br> Check for an index file in same directory as the BAM.<br><br>_Note: If a BAM input is missing an accompanying BAM index file in the same directory,<br> `validate` will not throw an exception but will print a warning._|
-| vcf | Validate vcf using `vcftools` |
-| fasta |  |
-| bed | |
-| py | |
+| File Type     | Description |
+| :-------:     | :---------: |
+| BAM           | Validate BAM/CRAM/SAM using `pysam`. <br> Check for an index file in same directory as the BAM.<br><br>_Note: If a BAM input is missing an accompanying BAM index file in the same directory,<br> `validate` will not throw an exception but will print a warning._|
+| VCF           | Validate vcf using `VCFtools` |
+| FASTA         | |
+| BED           | |
+| Python script | |
```

Comment:
Do we want to add descriptions?

Reply:
Maybe the column name is misleading here, the `Description` is meant to be what validations are done for that file type rather than an actual description of what that file type is